packages/plugins/rubygem-smart_proxy_pulp/rubygem-smart_proxy_pulp.spec

```
@@ -2,23 +2,24 @@
 %{?scl:%scl_package rubygem-%{gem_name}}
 %{!?scl:%global pkg_name %{name}}
 
+%{!?_root_bindir:%global _root_bindir %{_bindir}}
 %{!?_root_datadir:%global _root_datadir %{_datadir}}
 %{!?_root_localstatedir:%global _root_localstatedir %{_localstatedir}}
 %{!?_root_sysconfdir:%global _root_sysconfdir %{_sysconfdir}}
 
 %global gem_name smart_proxy_pulp
 %global plugin_name pulp
 
-%global foreman_proxy_min_version 1.25
+%global foreman_proxy_min_version 1.24
```

I don't think this is right. It is actually 2.3 now since it uses `rackup_path` introduced there:
```suggestion
%global foreman_proxy_min_version 2.3
```

---

Thats the default in the gem2rpm template, I'll adjust this plugin. (Fixing the template if we want is a separate issue)